building_sim_plugins/building_ignition_plugins/CMakeLists.txt

```diff
@@ -130,6 +131,43 @@ target_link_libraries(lift
     #${tf2_ros_LIBRARIES}
 )
 
+###############################
+# crowd simulator stuff
+###############################
+if (menge_FOUND)
+  add_library(crowd_simulator_ign
+    SHARED
+    src/crowd_simulator.cpp
+  )
+
+  target_include_directories(crowd_simulator_ign
+    PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+    ${building_sim_common_INCLUDE_DIRS}
+    ${rclcpp_INCLUDE_DIRS}
+    ${menge_INCLUDE_DIRS}
+    ${IGNITION-COMMON_INCLUDE_DIRS}
+  )
+
+  target_link_libraries(crowd_simulator_ign
+    PUBLIC
+    ignition-gazebo${IGN_GAZEBO_VER}::ignition-gazebo${IGN_GAZEBO_VER}
+    ignition-plugin${IGN_PLUGIN_VER}::ignition-plugin${IGN_PLUGIN_VER}
+    ${rclcpp_LIBRARIES}
+    ${building_sim_common_LIBRARIES}
+    ${menge_LIBRARIES}
+  )
```

Comment:
We should probably start moving the ignition plugins to `ament_target_dependencies`, the same as the other libraries.

Reply:
Changed that part in CMakelists